### DISCUSSION TITLE
一分钟读论文：《WebAssembly 与 JS 在移动设备上的能耗对比》

### DISCUSSION THREAD

**Discussion by giscus** [unbug/unbug.github.io#4] [Announcements]
# 一分钟读论文：《WebAssembly 与 JS 在移动设备上的能耗对比》

在真实应用中评估 WebAsssembly 更多关注的是 API 和性能，而移动设备显然也需要关注能耗。荷兰阿姆斯特丹自由大学的论文《Comparing the Energy Efficiency of WebAssembly and JavaScript in Web Applications on Android Mobile Devices》在 Android 上通过对 32 个 Web 应用程序基于 Firefox、Chrome 跑 WebAssembly 与 JavaScript 之间的能耗进行分析，结果发现 WebAssembly 的能耗显著低于 JavaScript。

https://unbug.github.io/Comparing-the-Energy-Efficiency-of-WebAssembly-and-JavaScript-in-Web-Applications-on-Android-Mobile-Devices/

**Comment by unbug** [comment]
Firefox 节能40%！Chrome 真的是电老虎，Google 做点什么吧
